velox/type/tests/TimestampTest.cpp

```diff
@@ -418,5 +418,71 @@ TEST(TimestampTest, tmToStringTimestamp) {
   // %FT%T - equivalent to "%Y-%m-%dT%H:%M:%S" (the ISO 8601 timestamp format)
   testTmToString("%FT%T", TimestampToStringOptions::Mode::kFull);
 }
+
+TEST(TimestampTest, leadingPositiveSign) {
+  TimestampToStringOptions options = {
+      .leadingPositiveSign = true,
+      .zeroPaddingYear = true,
+      .dateTimeSeparator = ' ',
+  };
+  auto testTmToString = [&](Timestamp ts, const std::string& expected) {
+    std::tm tm;
+    Timestamp::epochToUtc(ts.getSeconds(), tm);
+    ASSERT_EQ(Timestamp::tmToString(tm, ts.getNanos(), options), expected);
+  };
+
+  const std::vector<Timestamp> input = {
+      Timestamp(253405036800, 0), Timestamp(253402231016, 0)};
+
+  const std::vector<std::string> expected = {
+      "+10000-02-01 16:00:00.000000000", "9999-12-31 04:36:56.000000000"};
+
+  for (int32_t i = 0; i < input.size(); ++i) {
```

Comment:
This loop iterated over just 2 values. It might be more readable as 

ASSERT_EQ(testTmToString(Timestamp(253405036800, 0)), "+10000-02-01 16:00:00.000000000");

ASSERT_EQ(testTmToString(Timestamp(253402231016, 0)), "9999-12-31 04:36:56.000000000");